docs/flow-setup-tutorial.md

```diff
@@ -0,0 +1,34 @@
+# Flow setup Tutotrial
+
+Flow is a static-type checker for Javascript code
+
+## The steps to install Flow correctly from scratch are
+
+####  Installation tip : install flow in the root folder of zulip-mobile
+
+####  package installer used - yarn
+
+## Installing flow steps
+
+1. install babel compiler with command line interface,
+	run command <$yarn add --dev babel-cli babel-preset-flow>
+2. change the present from "react-native" to "flow"
+>	{
+>	"present": ["flow"]
```

Comment:
Why is that? We either should change it ourselves or not suggest it at all?

Reply:
The flow doc specifies to set the "present" variable to flow in the .babelrc , i checked the file and it had react-native present, i added flow as another value but it didn't worked so i experimented and replaced react-native with flow and it worked fine.

Reply:
The .babelrc file is generated after installing babel-cli so we need to give information to the developer about the change.

Reply:
update: it is working with react-native as it's value as well. Let me perform some other checks as well before final reporting.